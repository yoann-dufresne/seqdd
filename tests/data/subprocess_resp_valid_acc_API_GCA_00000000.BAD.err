  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   278  100   278    0     0   2770      0 --:--:-- --:--:-- --:--:--  2780
