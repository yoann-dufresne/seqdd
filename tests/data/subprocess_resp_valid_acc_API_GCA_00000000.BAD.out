<?xml version="1.0" encoding="UTF-8"?>
<ErrorDetails>
  <timestamp>1732876814771</timestamp>
  <status>400</status>
  <error>Bad Request</error>
  <message>Unknown accession format:GCA_00000000.BAD.</message>
  <path>/ena/browser/api/xml/GCA_00000000.BAD</path>
</ErrorDetails>
